src/main/web_src/src/components/bestillingsveileder/stegVelger/steg/steg3/BestillingInfoboks.tsx

```diff
@@ -27,14 +27,17 @@ export const BestillingInfoboks = ({ bestillingsdata }: BestillingInfoboks) => {
 		tpsfInfo &&
 		(tpsfInfo.spesreg === 'SPFO' ||
 			tpsfInfo.spesreg === 'SPSF' ||
+			tpsfInfo.spesreg === 'SFU' ||
```

Comment:
Kunne vi hatt en egen sjekk på SFU? For nå får vi linjen om kode 19 når vi har kode 6 eller 7 også, og det hadde vært nice om den bare synes når man har valgt kode 19 😄

Reply:
Ja sant, tenker denne kan fjernes igjen siden info dukker opp når man velger kode 19 😄

Reply:
Stemmer, det bør jo holde å vise den ett sted 😄